sanic/src/feature_extraction.py

```diff
@@ -0,0 +1,49 @@
+import cv2
+import mediapipe as mp
+
+mp_hands = mp.solutions.hands
+
+# Python Generator that yields an image every call
+def getImagesFromVideo(video):
+    videoWrapper = cv2.VideoCapture(video)
+
+    if videoWrapper.isOpened() == False:
+        raise Exception("Could not open video")
+    
+    while videoWrapper.isOpened():
+        wasReadSuccess, image = videoWrapper.read()
+
+        if wasReadSuccess == False:
+            break # End once video is done reading.
+
+        yield image
+        
+    videoWrapper.release()
+
+def getLandmarksFromVideo(video, flipped=False):
+
+    frames = []
+
+    with mp_hands.Hands(
+        model_complexity=0,
+        max_num_hands=2,
+        min_detection_confidence=0.5,
+        min_tracking_confidence=0.5) as handtracker:
+        
+        for image in getImagesFromVideo(video):
+            if flipped:
+                image = cv2.flip(image, 1)
+
+            result = handtracker.process(cv2.cvtColor(image, cv2.COLOR_BGR2RGB))
```

Comment:
wat is dis? hvad er det for noget color shizzle?

Reply:
Man ændre "color space" så det virker med mediapipe.. Hugget direkte fra mediapipe python example guide xd

Reply:
Så det er magi, got it xd

Reply:
/r/blackmagicfuckery